packages/cli/commands/upload.js

```diff
@@ -204,6 +205,24 @@ exports.handler = async options => {
       absoluteSrcPath,
       options.convertFields
     );
+
+    if (options.overwrite) {
+      //  If overwrite is true, will first delete the dest folder and then upload src. Cleans up files that only exist on HS.
+      const overwriteFile = await overwriteFilePrompt(dest);
```

Comment:
If we do go with prompting here I would want a `--force` option for CI/CD usages. But since `hs watch` doesn't prompt like this maybe we just shouldn't? @TanyaScales ?

Reply:
I'm slightly terrified of developers deleting their entire dest source because they mess something up locally -- I hear that we aren't prompting on watch. So, I am conflicted. If we were to remove prompting on each file -- which I do agree could be verbose in some cases -- I would almost want a confirmation prompt initially. 

`You are about to remove any remote files on your HubSpot account ${accountName} at ${filepath} that don't exist locally. Are you sure you want to do this? y/N`

^ Does that seem like a fair compromise?
---
Could also use a `--force` flag in this case for CI/CD usage